server/graphql/programs/src/queries/encounter.rs

```diff
@@ -7,14 +7,17 @@ use graphql_core::{
 use graphql_types::types::encounter::{
     EncounterConnector, EncounterFilterInput, EncounterNode, EncounterSortInput,
 };
-use repository::{EncounterFilter, PaginationOption};
+use repository::{EncounterFilter, EqualFilter, PaginationOption};
 use service::auth::{Resource, ResourceAccessRequest};
 
 #[derive(Union)]
 pub enum EncounterResponse {
     Response(EncounterConnector),
 }
 
+/// Returns a list of encounters.
+///
+/// Deleted encounters are excluded from the returned list.
```

Comment:
I think this is the expected behaviour (?) We could later have a filter flag to include deleted encounter (see note below)

Reply:
Can you give an example please?

Currently, the DELETED option is in the status filter but it is overwritten when used (could remove it from the graphql definition...)

I would have to look into the different status filter options like `equalTo`, `equalAny`, etc. to see if a user requested the DELETED encounter and only if they weren't requested filter out the DELETED encounters. Is this what you mean?

Reply:
I was meaning that we could add a `Deleted` option to this filter:

<img width="442" alt="image" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/f3ed2aa6-5437-48ff-8fab-f1ef80b4f9db">

though, as you say, it would require the DELETED status to be used in the query. 
No need to do anything, it was just a thought, of how we could implement the 'show me the list of deleted encounters'